### PR TITLE
refactor: rename and move interfaces JobAnalyzeStatus, JobAnalyzeInfo

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

- Moved page related type declaration into `page-directory/type.ts`
- Renamed page related interfaces. Removed 'model' from the name.

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
thx!!!
LGTM!

File: apps/web/src/services/asset-inventory/collector/model.ts [hunk 1]
```diff
@@ -147,15 +147,22 @@ export interface CollectorDeleteParameter {
...
-}
-export interface JobAnalyzeModel {
+    failure_tasks: number;
+    success_tasks: number;
     collector_id: string;
-    job_status: JobStatus[]
+    plugin_id: string;
+    created_at: string;
+    finished_at: string;
+    updated_at: string;
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/model.ts:167]: image

Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/model.ts:167]: image

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Pull request merged**